soda-core/src/soda_core/cli/cli.py

```diff
@@ -110,15 +110,13 @@ def _setup_contract_verify_command(contract_parsers) -> None:
         "-c",
         "--contract",
         type=str,
-        nargs="+",
-        help="One or more contract file paths. Use this to work with local contracts.",
+        help="One contract file path. Use this to work with local contracts.",
```

Comment:
NIT: I wouldn't emphasize the `one`
```suggestion
        help="Contract file path to verify. Use this to work with local contracts.",
```